wdl-grammar/CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## 0.17.0 - 09-15-2025
```

Comment:
Likewise, the only changes to `wdl-grammar` were `Cargo.toml` released (sorted and moved a dependency to the workspace dependencies table).

Reply:
same here